src/index.jsx

```diff
@@ -32,7 +33,8 @@ function createInitialState() {
   if (Object.keys(authenticationState).length === 0) {
     throw new Error('Empty authentication state returned from gettAuthenticationState()');
   }
-  return Object.assign({}, { configuration }, authenticationState);
+  const queryParameters = getQueryParameters();
```

Comment:
Again, probably not a problem for us right now, but downstream we should be aware that this won't actually update redux when the query params change.